client.go

```diff
@@ -51,6 +51,7 @@ import (
 const (
 	DefaultStorageURL               = "https://api.e3db.com"
 	DefaultEncryptedFileName        = "encrypted"
+	DefaultDownloadedFileName       = "downloaded"
```

Comment:
nit: the more I think about this the more I think we're better off using the record uuid or a random uuid with encrypted or downloaded prefixed (or postfixed) to it for these intermediate files